docs/learn/guides/database-integrations/mongodb_integrations/insert_data/ballerina.conf

```diff
@@ -0,0 +1,10 @@
+MONGO_HOST = "localhost"
```

Comment:
We can remove this file since we're giving instruction in the readme to add a conf file

Reply:
Fixed